components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -416,7 +468,7 @@ protected JWSAlgorithm mapSignatureAlgorithm(String signatureAlgorithm) throws I
      */
     protected JWTClaimsSet createJWTClaimSet(OAuthAuthzReqMessageContext authAuthzReqMessageContext,
                                              OAuthTokenReqMessageContext tokenReqMessageContext,
-                                             String consumerKey) throws IdentityOAuth2Exception {
+                                             String consumerKey, boolean isAccessToken) throws IdentityOAuth2Exception {
```

Comment:
method signature of protected methods should not be changed

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b